README.md

```diff
@@ -64,14 +74,173 @@ If you have a suggestion for a feature, please feel free to file it under "issue
 
-Of course, we'd prefer that if you have issues you [report them on GitHub](https://github.com/OfficeDev/script-lab/issues/new) instead, but you're free to leave any review comments you wish.
 
 <a id="contribute"></a>
-## Contribute code or samples to Script Lab
+## Contributing code or samples to Script Lab
 
 To contribute to Script Lab itself, or to contribute sample code, please see [CONTRIBUTING.md](CONTRIBUTING.md).
+
+
+
+<a id="frequently-asked-questions"></a>
+## Frequently Asked Questions (FAQs)
+
+
+<a id="import"></a>
+### How can I import someone else's snippet?
+
+Script Lab is built around sharing.  If someone gives you a URL to a GitHub GIST, simply open Script Lab, use the hamburger menu at the top left to see the menu, and choose "Import" category (either on the left or top, depending on the available screen space). Then, enter the URL of the GIST, and click the "Import" button at the bottom of the screen.  In just these few clicks, you will be able to view and run someone else's snippet!
+
+![Import](.github/images/import-snippet.jpg)
+
+
+<a id="external-libraries"></a>
+### How can I reference external libraries?
+
+One of the powerful things about Script Lab is the ability to incorporate other libraries with your snippets. This creates a bunch of fun and compelling scenarios (for example, see the color-picker example, which you can import using the GIST URL <https://gist.github.com/Zlatkovsky/c7ba602e8a107cfe5dd6c42ca41deac1>)
+
+![Import](.github/images/color-picker.jpg)
+
+The ease of importing varies depending on how the external library was created. If you are not sure, start with the first option, and progress to the next one if it doesn't work for you
+
+
+#### Easiest: library is hosted on a regular CDN, exposes a global variable, and/or is a UMD module
+
+This is the "happy path".  Simply reference the library (both the JS and the CSS, if relevant) in the "Libraries" tab.
+
+If the library is hosted on a CDN (much as `Office.js` is), you could reference it as a URL, as follows:
+
+    https://appsforoffice.microsoft.com/lib/1/hosted/office.js
+
+If the library is available as an NPM package, you can just reference it by name:
+
+    spectrum-colorpicker/spectrum.js
+    spectrum-colorpicker/spectrum.css
+
+This will use <https://unpkg.com/>, a global CDN for NPM libraries, to substitute in the appropriate files. Thus, the above is equivalent to:
+
+    https://unpkg.com/spectrum-colorpicker@1.8.0/spectrum.js
+    https://unpkg.com/spectrum-colorpicker@1.8.0/spectrum.css
+
+> Tip: To see the available files in a package, simply include a `/` after the package name, and you'll get a directory listing.  E.g., <https://unpkg.com/spectrum-colorpicker/>. It will also conveniently tell you the available version numbers, via a dropdown at the top right of the site:
+>
+> ![Import](.github/images/unpkg.jpg)
+
+
+#### Moderate: NPM package is an AMD module, but the library is also available via a "dist" build
```

Comment:
Wrong title for the given example. An AMD module is one that is loaded from require.js